lib/csv_importer.rb

```diff
@@ -69,8 +69,8 @@ def header
 
   # Initialize and return the `Row`s for the current CSV file
   def rows
-    csv.rows.map do |row_array|
-      Row.new(header: header, row_array: row_array, model_klass: config.model,
+    csv.rows.map.with_index do |row_array, index|
```

Comment:
Actually, it should be `.with_index(2)`, as the header line is not included in the rows array. The `report.invalid_rows` will never refer to an error on line 1 of the *file*. Agreed?